presto-main/src/main/java/com/facebook/presto/sql/gen/LambdaBytecodeGenerator.java

```diff
@@ -143,6 +149,52 @@ private static LambdaExpressionField defineLambdaMethodAndField(
         return new LambdaExpressionField(staticField, instanceField);
     }
 
+    public static BytecodeNode generateLambda(
```

Comment:
I don't understand why you are moving this method. I assume later commits will explain this.